tests/test_webhooks.py

```diff
@@ -122,3 +123,32 @@ def test_unrecognized_webhook_type_returns_untyped_webhook(
         )
         assert type(result).__name__ == "UntypedWebhook"
         assert result.dict() == json.loads(mock_unknown_webhook_body)
+
+    def test_validate_dsync_activated_event(self):
```

Comment:
It would be nice to generalize and parameterize a test like this, though we'd need all the input data.